NEWS.md

```diff
@@ -41,6 +41,8 @@ getting simplified output (#623) thanks @glaroc !
 
 ### BUG FIXES
 
+* `genbank2uid` now returns the correct ID when there are multiple possibilities
```

Comment:
don't add a news item in a version that's already on CRAN - can start a new section with a dev version number or so

Reply:
Oops, I assumed that the top section was the dev version. Should have checked that..